tests/unit/fshl/folds32.opt

```diff
@@ -1,4 +1,4 @@
-; TEST-ARGS: -root-only
+; TEST-ARGS: -root-only -smt-to:15000
```

Comment:
Right now this is necessary because memory always encodes disjointness between nonlocal blocks even if the src/tgt never accesses memory; this can be removed after the non_local_bid_upperbound is properly set.